pkgs/games/xivlauncher/default.nix

```diff
@@ -13,7 +13,7 @@ in
       owner = "goatcorp";
       repo = "XIVLauncher.Core";
       inherit rev;
-      hash = "sha256-HbOo1aCBYnLXI2QZEBSRQNchHD2/fo50M2ZnIXkRn6Y=";
+      hash = "sha256-nGCaVALgx3wNLgy77dxc7Am3zZjmi10kqSEhZi/WFyA=";
```

Comment:
can change to sha256-P1i12vw9KQOTL7QBxjKPSlda5rvoyl7DGmuuD5iVEQs=

Reply:
that's not the hash I get fwiw:

```console
$ nix-prefetch-github goatcorp XIVLauncher.Core --rev 1.0.6
{
    "owner": "goatcorp",
    "repo": "XIVLauncher.Core",
    "rev": "072aca4db439749e92fb189a2f762c57410fdda8",
    "hash": "sha256-NpdaPntaLrxf+4BH1d7kvDmjAuYqQJsTB+QHBDCm5fU="
}
```

Reply:
...argh ok I do get your hash from nix-build

currently busy raiding but this feels like a problem I've hit before, so ignore me

Reply:
For what it's worth I usually just stick in a gibberish hash, rebuild, then put in whatever it tells me 😅

Reply:
turns out forgetting to `--fetch-submodules` changes the hash :upside_down_face: